src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -274,6 +276,12 @@ export default {
 	},
 	mounted() {
 		this.updateLendFilterExp();
+		// check for newly assigned bounceback
+		const redirectFromUiCookie = cookieStore.get('redirectFromUi') || '';
+		if (redirectFromUiCookie === 'true') {
+			cookieStore.remove('redirectFromUi');
+			this.$router.push(this.getAlgoliaFilterUrl());
+		}
```

Comment:
This covers the scenario when visitor with no 'lend_filter" exp assignment clicks the 'filter' link on a new /lend-by-cat/some-cat page. At that time they are redirected to the legacy /lend/some-cat page and may be assigned into the experiment. If so they bounce back here so we catch that and display the /lend/filter page with query params for their intended filters applied. This is an auxillary behavior enabled by the new getAlgoliaFilterUrl method and mapping of old loan channel sets to new algolia facet params.

Reply:
Nice